src/app/challenges/challenge-details.tsx

```diff
@@ -28,6 +31,20 @@ const ChallengeDetail = () => {
   const color = data?.data.color ?? theme.colors.secondary[100];
   const { isOngoing } = isChallengeActive(data?.data?.end_at);
 
+  const handleShare = async () => {
+    if (data) {
+      const urlWithUserId = data.data.url.replace("[user_id]", loggedUser);
+      try {
+        await Share.share({
+          message: urlWithUserId, // Android
+          url: urlWithUserId, // iOS
```

Comment:
comment out this line, otherwise it sends the url x 2 on ios

Reply:
Fixed according to platform